checker/tests/index/ConstantArrays.java

```diff
@@ -0,0 +1,8 @@
+import org.checkerframework.checker.index.qual.*;
+
+class ConstantArrays {
+    void test() {
+        int[] b = new int[4];
+        @LTLengthOf("b") int[] a = {0, 1, 2, 3};
+    }
+}
```

Comment:
Please add more test cases.  Some that fail, some where the array is less than or equal to some array, somewhere it is less than or equal given some offset.